### DISCUSSION TITLE
SOMETHING WENT WRONG!

### DISCUSSION THREAD

**Discussion by b05-olaniyi** [iotaledger/firefly#2294] [Help & Support] [answered]
My Firefly Wallet keeps showing SOMETHING WENT WRONG and the network is not connected at all, in the first place I thought it was my PC

**Comment by t00k** [comment, marked as answer]
Can you go into Menu -> Firefly -> Error log and post what is shown there?

**Reply by b05-olaniyi** [reply]
Ok

**Reply by b05-olaniyi** [reply]
![IMG_20220207_012416_808.jpg](https://user-images.githubusercontent.com/99150549/152708131-ac28af44-dcd0-4e8d-88d6-ec30faaef5e0.jpg)

**Reply by t00k** [reply]
Do you use a VPN or similar?

First of all, please make sure you have your seed backed up, either 24 words/stronghold backup/ledger nano.
Then download Firefly 1.3.3 from firefly.iota.org if you are not already on latest version.
If you still have issues, log out -> add profile.

Then add a new profile, but use the same 24 words you already use right now, but on a new profile.

**Reply by b05-olaniyi** [reply]
No, I did not.


Okay, I will do just that 👌

**Reply by t00k** [reply]
Please remember to NEVER share any private keys with ANYONE. Scammers are lurking in this space. Trust nobody.
Hopefully the above steps will fix your issue. Good luck!

**Comment by Blick20** [comment]
Hi, unfortunately I have the same issue. I followed all the steps mentioned above. However, it is still not working, i.e., the netwoprk is still disconnected. I changed again the netwrok node, but without success. What else can I do?

**Reply by MiKeF05** [reply]
Do you use a VPN or a special firewall?

**Reply by b05-olaniyi** [reply]
hi, please what can we do now. i later discover that i formarted my pc?
